src/storybook-docs/installation.stories.mdx

```diff
@@ -1,27 +1,27 @@
-import { Meta, Story, Canvas } from '@storybook/addon-docs/blocks';
+import { Meta } from '@storybook/addon-docs/blocks';
 
-<Meta title="Instalasi" />
+<Meta title="Instalasi" author="Jabar Digital Service" />
 
 # Selamat Datang di Jabar Digital Service Sistem Desain
 
 **Jabar Digital Service Sistem Desain** adalah _open source_ JDS yang digunakan dalam mendesain dan mengembangkan aplikasi *web* maupun *mobile* dalam lingkungan pemerintahan Jawa Barat. Sistem desain ini terdiri dari komponen siap pakai dan kumpulan kode pendukungnya serta pedoman penggunaan untuk masing-masing komponen tersebut.
 
 ## Catatan
-Sistem desain ini masih diperuntukan digunakan pada *framework* VueJS selain itu kami juga berharap agar bisa mendukung penggunaan di *framework* lain dengan membuat *library* ini menjadi *web component* yang sedang kami kembangkan sampai saat ini.
+Sistem desain ini tidak disarankan untuk digabung dengan *UI Library* lain karena dapat menimbulkan desain yang tidak beraturan. Sistem desain ini juga masih diperuntukan digunakan pada *framework* VueJS selain itu kami juga berharap agar bisa mendukung penggunaan di *framework* lain dengan membuat *library* ini menjadi *web component* yang sedang kami kembangkan sampai saat ini.
```

Comment:
Perhaps that "_desain yang tidak beraturan_" can be better phrase?.

Something like, "_konflik antar definisi CSS_", or "_dapat menimbulkan perilaku komponen yang tidak diinginkan, terutama dari segi styling_".

Reply:
i don't know a better phrase for represents what i mean, but your advice is very helpful, I think I should replace it to "dapat menimbulkan perilaku komponen yang tidak diinginkan, terutama dari segi styling" is a better phrase than before @adrianpdm

Reply:
I have done change that phrase in https://github.com/jabardigitalservice/jds-design-system-vue-lib/pull/124/commits/f2ffe9a79709c2c40c261451ffb9a4ce4a28ee13

Reply:
Thank you :smile: